src/custom/utils/operator.ts

```diff
@@ -178,16 +178,10 @@ export type FeeQuoteParams = Pick<OrderMetaData, 'sellToken' | 'buyToken' | 'kin
   chainId: ChainId
 }
 
-export async function getFeeQuote({
-  sellToken,
-  buyToken,
-  amount,
-  kind,
-  chainId
-}: FeeQuoteParams): Promise<FeeInformation> {
-  const independentToken = kind === 'buy' ? buyToken : sellToken
+export async function getFeeQuote(params: FeeQuoteParams): Promise<FeeInformation> {
+  const { sellToken, buyToken, amount, kind, chainId } = params
```

Comment:
you really hate destructuring in the signature don't you

Reply:
Hehe, I love it, I just a super low threshold in which I move things away from the function definition

In this case was mostly motivated for printing all the params in the log